packages/ssi-types/src/types/vc.ts

```diff
@@ -66,7 +69,7 @@ export interface ICredentialStatus {
 export interface IIssuer {
   id: string
 
-  [x: string]: unknown
+  [x: string]: any
```

Comment:
why the change to any?

Reply:
Because you cannot simply access nested properties when it is unknown.

credentialSubject.mainProperty.subProperty